data/constructed-data/2025/20240429144820-construct-query-1-and-2.sparql

```diff
@@ -0,0 +1,32 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX org: <http://www.w3.org/ns/org#>
+
+CONSTRUCT {
+  ?vorigBestuursorgaanInTijd mandaat:bindingEinde "2025-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
+    ?bestuursorgaanInTijd a besluit:Bestuursorgaan ;
+      mandaat:isTijdspecialisatieVan ?bestuursorgaan ;
+      mu:uuid ?uuidInTijd ;
+      mandaat:bindingStart "2025-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
+  
+  # Mandaat gemeenteraadslid in gemeenteraad
+  ?mandaat a mandaat:Mandaat ;
+      mu:uuid ?uuid ;
+      org:role <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000011> .
+  
+  ?mandaatBestuursorgaanInTijd org:hasPost ?mandaat .
+} WHERE {
+  ?vorigBestuursorgaanInTijd mandaat:bindingEinde "2025-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
+      ?bestuursorgaanInTijd a besluit:Bestuursorgaan ;
+        mandaat:isTijdspecialisatieVan ?bestuursorgaan ;
+        mu:uuid ?uuidInTijd ;
+        mandaat:bindingStart "2025-01-01"^^<http://www.w3.org/2001/XMLSchema#date> .
+
+  # Mandaat gemeenteraadslid in gemeenteraad
+  ?mandaat a mandaat:Mandaat ;
+      mu:uuid ?uuid ;
+      org:role <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000011> .
+  
+  ?mandaatBestuursorgaanInTijd org:hasPost ?mandaat .
```

Comment:
i think we can get rid of the next file if we just remove the org:role restriction and replace mandaatBestuursorgaanInTijd by bestuursorgaanInTijd. This will get all created organen and their mandates (which seems to be the goal)

Reply:
and the other mandaat files too \o/